app/models/concerns/profile_tailoring.rb

```diff
@@ -2,6 +2,16 @@
 
 # Methods that are related to profile tailoring
 module ProfileTailoring
+  def clone_to(account:, policy:, os_minor_version: nil)
+    new_profile = in_account(account, policy)
+    new_profile ||= create_child_profile(account, policy)
+
+    # Update the os minor version if not already set
+    new_profile.update_os_minor_version(os_minor_version)
+
+    new_profile
+  end
```

Comment:
This isn't specific to profile tailoring. If possible, it'd be good to keep it in profile.rb and move other things out to keep the class length short enough for rubocop.